test/functional/web/safari-ssl-e2e-specs.js

```diff
@@ -1,125 +1,140 @@
-//       try {
-//         driver = await initSession(caps);
-//         await driver.source().should.eventually.include('Arbitrary text');
-//         await driver.quit();
-//         await B.delay(1000);
-
-//         // Now do another session using the same cert to verify that it still works
-//         // (Don't do it on CLOUD. Restarting is too slow)
-//         if (!process.env.CLOUD) {
-//           await driver.init(caps);
-//           await driver.get(LOCAL_HTTPS_URL);
-//           await driver.source().should.eventually.include('Arbitrary text');
-//         }
-//       } finally {
-//         await deleteSession();
-//       }
-//     });
-
-//     describe('cookies', function () {
-//       const secureCookie = Object.assign({}, newCookie, {
-//         secure: true,
-//         name: 'securecookie',
-//         value: 'this is a secure cookie',
-//       });
-
-//       before(async function () {
-//         driver = await initSession(caps);
-//       });
-
-//       beforeEach(async function () {
-//         await driver.get(LOCAL_HTTPS_URL);
-//         await driver.setCookie(oldCookie1);
-//         await driver.deleteCookie(secureCookie.name);
-//       });
-
-//       it('should be able to set a secure cookie', async function () {
-//         let cookies = await driver.allCookies();
-//         doesNotIncludeCookie(cookies, secureCookie);
-
-//         await driver.setCookie(secureCookie);
-//         cookies = await driver.allCookies();
-
-//         doesIncludeCookie(cookies, secureCookie);
-
-//         // should not clobber old cookie
-//         doesIncludeCookie(cookies, oldCookie1);
-//       });
-//       it('should be able to set a secure cookie', async function () {
-//         await driver.setCookie(secureCookie);
-//         let cookies = await driver.allCookies();
-
-//         doesIncludeCookie(cookies, secureCookie);
-
-//         // should not clobber old cookie
-//         doesIncludeCookie(cookies, oldCookie1);
-
-//         await driver.deleteCookie(secureCookie.name);
-
-//         cookies = await driver.allCookies();
-//         doesNotIncludeCookie(cookies, secureCookie);
-//       });
-//     });
-//   });
-// }
+import chai from 'chai';
+import os from 'os';
+import https from 'https';
+import chaiAsPromised from 'chai-as-promised';
+import B from 'bluebird';
+import { MOCHA_TIMEOUT, initSession, deleteSession, hasDefaultPrebuiltWDA } from '../helpers/session';
+import {
+  doesIncludeCookie, doesNotIncludeCookie, newCookie, oldCookie1
+} from './helpers';
+import { SAFARI_CAPS, amendCapabilities } from '../desired';
+
+
+const pem = B.promisifyAll(require('pem'));
+
+chai.should();
+chai.use(chaiAsPromised);
+
+const HTTPS_PORT = 9762;
+
+const LOCAL_HTTPS_URL = `https://localhost:${HTTPS_PORT}/`;
+
+let caps;
+let pemCertificate;
+
+if (!process.env.REAL_DEVICE && !process.env.CLOUD) {
```

Comment:
shall we deprecate these env vars in this PR?

Reply:
done as https://github.com/appium/appium-xcuitest-driver/pull/1507/commits/b7ef6778c0d592bcb8d909c0a2e43aa140f06684 and https://github.com/appium/appium-xcuitest-driver/pull/1507/commits/4ae2757ff59d8b985a0c0115c98c48eec5ad4afc